frontend/src/pages/DashboardPage.tsx

```diff
@@ -1,17 +1,39 @@
-import { Container, Skeleton } from "@mui/material";
+import { Grid, Typography } from "@mui/material";
+
+import { BarChart } from "../components/charts/BarChart";
+import { PieChart } from "../components/charts/PieChart";
 
 export const DashboardPage = () => {
   return (
-    <Container
-      sx={{
-        height: "100vh",
-        width: "100%",
-        display: "flex",
-        alignItems: "center",
-        justifyContent: "center",
-      }}
-    >
-      <Skeleton variant="rounded" width="50%" height="30%" />
-    </Container>
+    <Grid container>
```

Comment:
Leave the code here first. Am in the midst of figuring out how to better structure the dashboard. I am trying to do something like this.
![image](https://user-images.githubusercontent.com/65804451/189728580-7e5efddd-26d7-4131-a179-6d4f24b9fdba.png)

However, the chart comes in different sizes so it is hard to contain them in grids